source/CleoBase.cpp

```diff
@@ -6,6 +6,7 @@ namespace CLEO
 {
     CCleoInstance CleoInstance;
     CCleoInstance& GetInstance() { return CleoInstance; }
+    std::stack<int>         CleoStack;
```

Comment:
Seems like it should hold SCRIPT_VAR type. Also what with the strings?

Reply:
strings are out of scope for now